worker-languagedetection-container/pom.xml

```diff
@@ -303,7 +303,7 @@
                             <alias>worker-languagedetection</alias>
                             <name>${dockerCafDataProcessingOrg}worker-languagedetection${dockerProjectVersion}</name>
                             <build>
-                                <from>${projectDockerRegistry}/cafapi/opensuse-jre21</from>
+                                <from>${homeDockerPrereleaseRegistry}/cafapi/oraclelinux-jre21:1</from>
```

Comment:
You shouldn't need any changes here, this is all controlled on the main pom now.